advanced/firmware/src/bin/usb-1.rs

```diff
@@ -43,10 +43,8 @@ fn on_event(_usbd: &USBD, event: Event) {
         Event::UsbReset => todo!(),
 
         Event::UsbEp0DataDone => todo!(),
+        // leave this at it is for now.
 
-        Event::UsbEp0Setup => {
-            log::info!("goal reached; move to the next section");
```

Comment:
I would leave this `info!` in the starter code. Otherwise it's a bit harder to realize you have completed this section.